canonical/maas-react-components
# docs: add storybook

## Done
- setup storybook
- setup storybook publish action to github pages
- add link to live storybook to the main README.md file 
- update README.md main section style



## QA steps

- Run `npm run storybook` and make sure it loads (will be empty)
- Verify that the [docs publish github action](https://github.com/canonical/maas-react-components/actions/workflows/publish-docs.yml) run successfuly
- Verify that an empty storybook can be accessed by visiting <https://canonical.github.io/maas-react-components>



## Fixes

Fixes: 



## Screenshots
![Google Chrome screenshot 000954@2x](https://github.com/canonical/maas-react-components/assets/7452681/8c89d9fd-8bee-44bf-810f-803610b38472)

![Google Chrome screenshot 000958@2x](https://github.com/canonical/maas-react-components/assets/7452681/81458712-529a-42fb-9c9e-64c6cf0d1e63)

## Notes
